src/bsoncxx/view_or_value.hpp

```diff
@@ -66,14 +66,14 @@ class view_or_value {
     /// @param value
     ///   A Value type.
     ///
-    BSONCXX_INLINE view_or_value(Value&& value) : _value{std::move(value)}, _view{_value->view()} {
```

Comment:
I don't understand this change. Would you mind explaining it to me? Also, how were we able to take _value->view() before? I don't see how we know this is defined.